Source/WebCore/PAL/pal/spi/mac/ScreenCaptureKitSPI.h

```diff
@@ -109,18 +115,18 @@ typedef NS_OPTIONS(NSUInteger, SCContentSharingAllowedPickerModes) {
     SCContentSharingPickerAllowedModeSingleDisplay         = 1 << 3,
 };
 
-@interface SCContentSharingPickerConfiguration : NSObject
+@interface SCContentSharingPickerConfiguration (Staging_110281298)
```

Comment:
Not sure what this change means.

Reply:
It is declaring the interfaces for both internal and open source builds, so we don't have to coordinate updating all of the bots to the new SDK.